lib/mayaUsd/resources/scripts/mayaUsdUtils.py

```diff
@@ -71,3 +75,80 @@ def isPulledMayaReference(dagPath):
 
     _, _, _, prim = getPulledInfo(dagPath)
     return prim and prim.GetTypeName() == 'MayaReference'
+
+
+def getMonoFormatFileFilterLabels(includeCompressed = True):
+    """
+    Returns a list of file-format labels for individual USD file formats.
+    This list is not directly usable in the dialog command, they would need
+    to be join with ';;'. Call getUSDDialogFileFilters() instead as it does
+    the joining.
+    """
+    if includeCompressed:
+        labelAndFilters = [
+            ("kUsdASCIIFiles", "*.usda"),
+            ("kUsdBinaryFiles", "*.usdc"),
+            ("kUsdCompressedFiles", "*.usdz"),
+        ]
+    else:
+        labelAndFilters = [
+            ("kUsdASCIIFiles", "*.usda"),
+            ("kUsdBinaryFiles", "*.usdc"),
+        ]
```

Comment:
To avoid duplication here couldn't it initialize the labelAndFilters to usda/usdc and then if includeCompressed add the usdz?